frontend/src/user/pages/ViewProfile/ViewProfile.tsx

```diff
@@ -75,35 +102,296 @@ export const ViewProfile = () => {
+                          {memb.year}
+                        </Fragment>
+                      ))
+                    : '-'}
+                </td>
+              </tr>
+              <tr>
+                <th>EYCA nebo členská karta</th>
+                <td>
+                  {user.eyca_card?.valid_till ? (
+                    'ano, platnost do: ' +
+                    formatDateTime(user.eyca_card?.valid_till)
+                  ) : (
+                    <>
+                      ne <br />
+                      <ExternalButtonLink
+                        tertiary
+                        small
+                        target="_blank"
+                        href="https://www.eyca.cz/"
+                      >
+                        výhody EYCA evropské karty mládeže
+                      </ExternalButtonLink>
+                    </>
+                  )}
+                </td>
+              </tr>
+              <tr>
+                <th>Datum vzniku tvého profilu v BIS</th>
+                <td>
+                  {user.date_joined ? formatDateTime(user.date_joined) : '-'}
+                </td>
+              </tr>
             </tbody>
           </table>
-        </section> */}
+          <div className={styles.buttonGroup}>
+            <ButtonLink primary to="/user/akce/zucastnene">
+              Akce, kterých jsem se zúčastnil/a
+            </ButtonLink>
+            <ButtonLink primary to="/org/akce/vsechny">
+              Akce, které jsem organizoval/a
+            </ButtonLink>
+          </div>
+          <InfoBox>
+            <p>
+              Jako organizátor Hnutí Brontosaurus můžeš pro pořádání akcí
+              využívat podporu a zázemí HB. Zde najdeš užitečné informace a
+              pořádání akcí, které jsou dostupné pouze pod heslem
+            </p>
+            <ul>
+              <li>
+                <a
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  href="https://mozek.brontosaurus.cz/"
+                  className={styles.help}
+                >
+                  Mozek HB
+                </a>
+                , kde jsou všechny důležité dokumenty. Přihlašovací jméno:
+                brontosaurus, heslo: Brontosaurus40
+              </li>
+              <li>
+                <a
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  href="https://www.canva.com/ "
+                >
+                  Canva
+                </a>
+                , kde můžeš využívat předplacenou Canvu Pro pro vytváření
+                plakátů a dalších propagačních materiálů. Přihlašovací email:
+                marketing@brontosaurus.cz, heslo: jedemedal24
+              </li>
+              <li>
+                Připravenou{' '}
+                <a
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  href="https://zpetna-vazba.brontosaurus.cz/login.php"
```

Comment:
Is the same link with just slightly different text really supposed to be there twice?

Reply:
yes, it should be. Link is the same, but there are two separate actions that need to be done.